### DISCUSSION TITLE
vscode cpu100%

### DISCUSSION THREAD

**Discussion by yaolunmao** [vuejs/language-tools#504] [Q&A] [answered]
想问一下各位出现过这个问题吗？只要我的代码安装完node_modules,也就是`npm i`，vscode的cpu就会飙升到100%，代码提示啥的都没了，把volar禁用了就没事了，不知道是和node_modules里面的哪个包冲突了
以下是机翻，英语不好最近在学，请见谅

Do you have this question? As long as my code installs node_ Modules, that is, `npm i`, the CPU of vscode will soar to 100%. There are no code prompts. It's OK to disable volar.I don't know which package conflicts with node_modules

**Comment by yaolunmao** [comment, marked as answer]
是因为这两个包
`    
"@vicons/fluent": "^0.11.0",
    "naive-ui": "^2.18.2"
,`

**Reply by johnsoncodehk** [reply]
See https://github.com/johnsoncodehk/volar/issues/507#issuecomment-922251114.
